infra/base-images/base-runner/dataflow_tracer.py

```diff
@@ -85,14 +85,17 @@ def collect_traces(binary, corpus_dir, dft_dir):
       'failed': 0,
   }
 
+  files_to_trace = {}
```

Comment:
nit: rename to `trace_sizes` or something similar that conveys this is for tracking sizes?

Reply:
the point was that these are the files we don't skip, i.e. the ones we would trace with DFSan. Renamed to `files_and_sizes` to be (hopefully) more clear